### ISSUE TITLE
`albumentations==1.4.5` UItralytics ValueError: Key bboxes is not in available keys.

### ISSUE THREAD

**Issue by glenn-jocher** [albumentations-team/albumentations#1699]
## Describe the bug

There appears to be a bug or an issue when using the latest albumentations release from today with Ultralytics, with CI showing the following errors. We've pinned albumentations to <=1.4.4 temporarily to address this.

See https://github.com/ultralytics/ultralytics/actions/runs/8943790283/job/24569265731

```
    def _check_args(self, **kwargs: Any) -> None:
        checked_single = ["image", "mask"]
        checked_multi = ["masks"]
        check_bbox_param = ["bboxes"]
        check_keypoints_param = ["keypoints"]
        shapes = []
        for data_name, data in kwargs.items():
            if data_name not in self._available_keys and data_name not in ["mask", "masks"]:
                msg = f"Key {data_name} is not in available keys."
>               raise ValueError(msg)
E               ValueError: Key bboxes is not in available keys.

/opt/conda/lib/python3.10/site-packages/albumentations/core/composition.py:313: ValueError
```

<img width="1952" alt="Screenshot 2024-05-03 at 22 48 23" src="https://github.com/albumentations-team/albumentations/assets/26833433/9915641c-7322-4122-a1b6-40bab8710a80">

### To Reproduce

```bash
# Install latest version
pip install ultralytics albumentations==1.4.5

# Train a YOLO model
yolo train
```

**Comment by ternaus** [comment]
Thanks! Will look in the issue ASAP

**Comment by ternaus** [comment]
@glenn-jocher 

Fixed in https://github.com/albumentations-team/albumentations/pull/1702

Released version 1.4.6 with issue being fixed.

**Comment by glenn-jocher** [comment]
@ternaus awesome, thank you!
